src/features/list-view/__tests__/ListView-test.js

```diff
@@ -49,11 +43,29 @@ describe('features/list-view/ListView', () => {
                 rowIndex,
             });
 
-            const testCell = shallow(cell);
+            expect(cell.props.children[0]).toBe(cellData);
+            expect(cell.props.className).toBe(className);
+        });
 
-            expect(testCell.text()).toEqual(cellData);
+        test.each`
```

Comment:
Let's consider merging back into its previous form. There's some copy / paste that I'd like to avoid.